content/md/productivity/manage-your-rules.md

```diff
@@ -102,8 +103,10 @@ Depending on the action you choose, the template changes. The table below lists
 | Clear categories  | Delete all the categories of your product/product model selection  |
 | Copy attribute value  | Copy the value from an attribute to another one  |
 | Remove categories  | Remove the categories of your product/product model selection  |
-| Set attribute value  | Set value to a multi select, number or boolean attribute. It will remove the former values |
+| Add attribute value  | Add values to a multi select, simple select, number, text, price collection or boolean attribute. It will keep the former values |
```

Comment:
"add" action only works with multi-select, price collections, asset collections and reference entity multiple link